### PR TITLE
added device orientation

### PR DESCRIPTION
**Pull request by melihberberolu**

### CODE REVIEW

File: lib/useDeviceOrientation.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+  useEffect(
+    () => {
+      Dimensions.addEventListener('change', onChange);
+
+      return () => {
+        Dimensions.removeEventListener('change', onChange);
+      };
+    },
+    [orientation.portrait, orientation.landscape]
```
Comment (melihberberolu) [inline lib/useDeviceOrientation.js]: Can i give orientation object for skipping effects. Did you know? @dabit3

Comment (dabit3) [reply lib/useDeviceOrientation.js]: Hmm, not sure, but this is a nice addition

Comment (dabit3) [reply lib/useDeviceOrientation.js]: @melihberberolu lmk when you're ready for me to merge

**Comment by melihberberolu** [comment]
Merge please :)

**Comment by dabit3** [comment]
@melihberberolu  💯💯💯💯

**Pull request merged**